docs/user/lti.rst

```diff
@@ -0,0 +1,232 @@
+     - ✔
+     - ✔
+     - ✔
+   * - Quiz exercise
+     - ✔
+     - ✔
+     - ✔
+   * - Modeling exercise
+     - ✔
+     - ✔
+     - ✔
+   * - Text exercise
+     - ✔
+     - ✔
+     - ✔
+   * - File Upload exercise
+     - ✔
+     - ✔
+     - ✔
+
+Prerequisite for Server Admins
+-------------------------------
+Before instructors integrate Artemis or students access and start Artemis exercises through Moodle or any other LMS via LTI, server administrators must ensure that LTI is active with the necessary profile on the server. If you're a server admin, please make sure to configure this setting to allow instructors and students to utilize the LTI integration.
+
+LTI 1.3 Instructor Guide
+---------------------------
+Instructors have the capability to seamlessly integrate the Artemis platform with other Learning Management Systems (LMS) like Moodle using the LTI 1.3 standard. The process is straightforward and involves a few key steps:
+
+#. Connecting Artemis to Other Platforms: Begin by enabling the Online Course setting in Artemis, which activates the LTI features.
+#. Retrieve Dynamic Registration URL: Once the Online Course setting is enabled, you can obtain the Dynamic Registration URL from Artemis, which is essential for the integration process.
+#. Establish Configuration in Moodle: With the Dynamic Registration URL in hand, proceed to your Moodle platform and input this URL to establish a connection between Artemis and Moodle.
+#. Link Specific Exercises: Instructors can also link individual exercises from Artemis directly to their Moodle courses, ensuring students have direct access to specific assignments.
+
+By following these steps, instructors can effortlessly bridge the gap between Artemis and Moodle, providing students with a unified learning experience.
+
+1. Connecting Artemis to Other Platforms (e.g., Moodle)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Instructors can connect Artemis online courses via LTI to other Learning Management Systems (e.g., Moodle, edX).
+Below given steps for how to configure Artemis over LTI with Moodle.
+
+1.1 Enable the Online Course Setting in Artemis:
+"""""""""""""""""""""""""""""""""""""""""""""""""
+Before diving into the LTI configuration, it's crucial to ensure that the Online Course setting is enabled in Artemis. This setting activates the LTI configurability, allowing instructors to link Artemis with Moodle or other LMS platforms.
+To enable the Online Course setting in Artemis, follow the steps below:\
+
+#. Access Course Management: Start by logging into your Artemis account. Locate and click on the |course-management| option.
+#. Select the Desired Course: From the list of available courses, navigate to the course you wish to configure for LTI integration.
+#. Edit Course Settings: On the course overview page, you'll find an |course_edit| button, located at the top right corner. Clicking this will allow you to modify various course settings.
```

Comment:
```suggestion
#. Edit Course Settings: On the course overview page, you will find an |course_edit| button, located at the top right corner. Clicking this will allow you to modify various course settings.
```